### ISSUE TITLE
Test suite report renders inside another

### ISSUE THREAD

**Issue by tomas-hartman** [Hargne/jest-html-reporter#105]
Test suite report is incorrectly rendered inside of another. 

**Screenshot**: [here](https://user-images.githubusercontent.com/44980733/82222531-b3ef5b00-9921-11ea-8910-05e285c06638.png)

**Test run with params**:

`"jest ./test/e2e/ --maxWorkers=2 --no-cache --config=./test/e2e/jest.config.js --ci"`

**Reporters settings**:

```js
[
  'default',
  ['../../node_modules/jest-html-reporter',
     {
      pageTitle: 'E2E tests Seznam Browser',
      outputPath: 'coverage/report.html',
      dateFormat: 'dd. mm. yyyy HH:MM:ss',
      includeConsoleLog: true,
      includeFailureMsg: true,
     },
  ],
]
```

**Backend:**
- jest-html-reporter: 3.1.0
- jest: 26.0.1
- node.js: 13.8.0

**Comment by Hargne** [comment]
It looks like there might be due to unescaped characters in the test description/name that breaks the report structure.

**Comment by tomas-hartman** [comment]
Well some of my descriptions/test names are ending with a `)`, but no other 'dangerous' chars are used on my side if that's the point.

**Comment by Hargne** [comment]
Are you able to send over the title and description of the test in which this happens so I can try to reproduce this issue? :)

**Comment by tomas-hartman** [comment]
There you go. There are basically three types of titles/descriptions I use:

1. ending with `)` (both describe and tests)
```
describe('Login promo (TC390f)', () => { ... }
```

2. generated with each
```
test.each([
  ['TC390d-5', 'Oblíbené', 'bookmarks.html', 'bookmarks', 'html.bookmarks'],
  ['TC390d-6', 'Historie', 'hist.html', 'history', 'html.history'],
])('%s: %s', async (tcName, expected, iframeSrc, buttonClass, iframeHTMLClass) => { ... }
```

3. classical `it` block

```
it('TC390c-4: Header: trending topics are clickable', async () => { ... }
```

Titles throughout the test file (inside which the other test is inset - addressbar.test.js) are following:

```
- 'TC265: should write a string into addressbar & find it with Seznam.cz, found string should be displayed in addressbar'
- 'TC264: měl bych přejít na web, který zadám do addressbaru, nezávisle na tom, že je adresa bez http(s)'
- 'TC517: [bugshooting] emailovou adresu vyhledáme jako řetězec, nepřecházíme na ni jako na url'
- 'TC___: nexistující url a url-like looking stringy vyhledáme'
- 'TC536: Načítání adresy: adresa se po refocusu a enteru znovu načte'
```

**Comment by Hargne** [comment]
Thanks for the information.
I have run several tests with the same inputs and methods as you provided, but it all worked fine. This brings me to believe that it perhaps could be related to the actual functions being tested. I can see in the test descriptions that the methods you are testing are using html entities and iframes - are you able to try to run the same tests but with mocked methods instead (so that we can try to pinpoint where the issue appears)?

**Comment by tomas-hartman** [comment]
I guess I found the problem. It only happens (to me) in a file where there are no `describe` blocks, only some standalone `it` blocks in the 'root' scope of the file. When I wrapped them into a `I'm trying this` `describe` block, it works just fine, so I guess the reporter failed when trying to push content to `div.test-suitename`.

![obrazek](https://user-images.githubusercontent.com/44980733/82343233-def5af80-99f2-11ea-91fc-955d2965b974.png)

**Comment by Hargne** [comment]
Great catch! I will try to patch this

**Comment by tomas-hartman** [comment]
Great, thanks! :)

**Comment by Hargne** [comment]
There is now a fix for this in [v3.1.3](https://github.com/Hargne/jest-html-reporter/releases/tag/3.1.3)
